### PR TITLE
update documentation on metadata for external modules

### PR DESCRIPTION
**Pull request by boegel**
following https://github.com/hpcugent/easybuild-framework/pull/1637 and https://github.com/hpcugent/easybuild-framework/pull/1638

preview available at http://boegel-eb.readthedocs.org/en/metadata_prefix/Using_external_modules.html

@gppezzi, @pforai: please review?

### CODE REVIEW

File: docs/Using_external_modules.rst [hunk 1]
```diff
@@ -45,6 +50,18 @@ Format::
   key1 = value1
   key2 = value2, value3
 
+Default
+~~~~~~~
+
+Up until EasyBuild v2.7.0, no metadata for external modules was available by default.
```
Comment (gppezzi) [inline docs/Using_external_modules.rst]: maybe "Up until v2.6.0" ?

**Comment by gppezzi** [comment]
looks good

File: docs/Using_external_modules.rst (hunk 1 above)
Comment (boegel) [inline docs/Using_external_modules.rst]: good point, fixing & merging

**Comment by boegel** [comment]
Thanks for the review @gppezzi!

**Pull request merged**